frontend/src/App.js

```diff
@@ -1,23 +1,6 @@
-import logo from './logo.svg';
-import './App.css';
-
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
```

Comment:
deleting this stuff wasn't part of the task, people might be confused that there is no content of the page...